plugins/compliments/plugin/commands/comp_cmd.rb

```diff
@@ -0,0 +1,73 @@
+module AresMUSH
+  module Compliments
+    class CompGiveCmd
+
+      #comp <name>=<text>
+      include CommandHandler
+      attr_accessor :comp, :scene_or_names, :scene_id, :scene, :target_names
+
+      def parse_args
+        args = cmd.parse_args(ArgParser.arg1_equals_arg2)
+        self.scene_or_names = args.arg1
+        self.comp = args.arg2
+      end
+
+      def required_args
+        [ self.scene_or_names, self.comp ]
+      end
+
+      def handle
+        targets = []
+        if (self.scene_or_names.is_integer?)
+          self.scene_id = self.scene_or_names.to_i
```

Comment:
Better!  Still a few little wobbles here.

- A few lingering puts statements should probably be removed.
- It's still returning error messages that won't get handled anywhere.  You need to do `client.emit_failure` before you return.
- Line 26 has no translation string - which is fine by me, but since everything else was translated I thought maybe it was just missed.

Reply:
Fixed